examples/workbooks/Products&Reviews.ts

```diff
@@ -0,0 +1,527 @@
+import * as FF from "@flatfile/configure";
+import { FlatfileRecord } from "@flatfile/hooks";
+import { v4 as uuidv4 } from "uuid";
```

Comment:
add a comment that this uuid package would need to be installed

Reply:
❔ Isn't that what the import statement says?